tests/richenum/test_rich_enums.py

```diff
@@ -147,3 +147,18 @@ def test_unicode_handling(self):
         self.assertEqual(str(poop_okra), "Okra💩")
         if not PY3:
             self.assertEqual(unicode(poop_okra), u"Okra💩")
+
+    def test_specific_lookup_error_is_caught(self):
+        with self.assertRaises(Vegetable.LookupError):
+            Vegetable.lookup('canonical_name', 'meat')
+
+    def test_other_specific_lookup_error_is_not_caught(self):
+        class Meat(RichEnum):
+            COW = RichEnumValue("cow", "Cow")
+
+        try:
```

Comment:
Yeah, but this test is a bit more fragile as depends on the ordering of the `except` clauses.
It's also testing exception behavior documented in Python which doesn't need to be tested (unless the behavior changes between Python versions).

From the Python 2.7.11 docs:

> For an except clause with an expression, that expression is evaluated, and the clause matches the exception if the resulting object is “compatible” with the exception. An object is compatible with an exception if it is the class or a base class of the exception object, or a tuple containing an item compatible with the exception.

https://docs.python.org/2/reference/compound_stmts.html#except

So to check that a `Meat.LookupError` won't be caught by a `<NonMeat>.LookupError`, checking the exception's hierarchy is sufficient.